cluster-up/cluster/okd-4.1.2/provider.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 #This will get overwritten when run make generate, this will not be needed once https://github.com/kubevirt/kubevirtci/pull/126 is merged.
-image="okd-4.1@sha256:d452e8f910bd08b4aabe2a9b8fd82dc5984a3e95f7096b3ebd6c8ba836a5361d"
+image="okd-4.1@sha256:84ab0dcc301b1e5c801de4f30e9950641f65a39a8756663d02da6de70e22e19e"
```

Comment:
isn't `cluster-up` supposed to be "vendored" in?

Reply:
yes, it is, but this is a test that the image I build for kubevirtci actually works. I am going to abandon this (or maybe merge, and once the kubevirtci PR is merged create a new PR that vendors that) if it works, and give the nod to kubevirtci that it works.

Reply:
okay may way to add a /hold for this then

Reply:
Okay kubevirtci is updated, I will do a proper vendoring instead of this.